APMrover2/mode_guided.cpp

```diff
@@ -147,3 +159,44 @@ void ModeGuided::set_desired_turn_rate_and_speed(float turn_rate_cds, float targ
     // log new target
     rover.Log_Write_GuidedTarget(_guided_mode, Vector3f(_desired_yaw_rate_cds, 0.0f, 0.0f), Vector3f(_desired_speed, 0.0f, 0.0f));
 }
+
+void ModeGuided::stop_vehicle_with_loiter()
+{
+    if (!rover.is_boat() || !_reached_destination) {
+        return;
+    }
+
+    // get distance (in meters) to destination
+    _distance_to_destination = get_distance(rover.current_loc, _destination);
+
+    // if within loiter radius slew desired speed towards zero and use existing desired heading
+    if (_distance_to_destination <= g2.loit_radius) {
```

Comment:
I do not have any projects that abbreviate loiter as loit.
I think that loiter is good. Expressing the same word with multiple abbreviations does not make sense.